### DISCUSSION TITLE
ProHD internal tuner UK HD channels

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#554] [Q&A]
Hi guys

Hopefully a simple question.

Will the ProHD tuner scan for and pick up HD channels in the UK ? 

(My aerial needs replacing/realigning so I can't see many channels at all currently therefor unable to test this myself.)

I've read that the ProHD has a DVB-T tuner ( not  a DVB-T2 tuner ) if that helps.

Thanks,
Steve

**Comment by BobC1954** [comment]
Works perfectly, but must have .420 firmware installed otherwise it will hang up and crash during tuning.

**Reply by mike-uk** [reply]
Can you tell me where I can get .420 firmware, please?

**Comment by BobC1954** [comment]
Here it is. Need to unzip it and it should produce a .bin file that you place into Slingplayer folder.
[Uploading SbPbProHdFwV2_01_420_HwV1_0_0Current.zip…]()

**Reply by mike-uk** [reply]
Thanks you its updated now. Which sling player version do you use usually use is 1.5.1.343 all you need to setup?

**Reply by BobC1954** [reply]
Yes, .v1.5 works ok for setup.

**Reply by sp222222** [reply]
I have the "hybrid" version for the NA proHD ATSC digital channels. I thought the unbricking guide had the how to on scanning for digital ones, but I've flat forgotten how to do that (mine's been running great for months until the reset button died on it).  I need to rescan the ASTC channels into the PRoHD. So use the 1.5.1x player and then what?

**Reply by mike-uk** [reply]
You need 1.5.1.338 for US we are in UK so differet version, if it just rescan just reinstall and rescan in directory, edit and setup video. Its not the same as unbrick.

https://github.com/GerryDazoo/Slinger/files/11069720/Slingbox.ProHD.PC.Unbricking-Setup-Network-Guide.pdf

**Reply by sp222222** [reply]
take the 2.0 spm and add an IP listed entry. then scan the digital from it. this is how USA ATSC gets scanned in. I have my 57 channels back :)

**Comment by gbjbasdw** [comment]
Bob1954 - I know it is probably me being stupid but how do I get the .420 zip downloaded ? I've tried clicking on it while signed in but it just opens the thread again.

**Reply by mike-uk** [reply]
Actually, I found it online soon after I posted it and saw his message just after I updated it haha. 

I found it here
http://autoupdate.sling.com/wbsp/100.07.NA.99/

I have another problem now, its working but the picture just drop after few seconds and there is no sound. I wonder if my orange pi zero is too weak but its say only running at 10% load. Also what remote do you use for dvb-t?

**Reply by BobC1954** [reply]
Ah! I don't know, I simply dragged the file into the window as per what it says below.
I'll try again....

[SbPbProHdFwV2_01_420_HwV1_0_0Current.zip](https://github.com/GerryDazoo/Slinger/files/11069442/SbPbProHdFwV2_01_420_HwV1_0_0Current.zip)

**Reply by BobC1954** [reply]
I ran into the no sound issue after initial tuning. Then I added a composite input (nothing plugged in, just a dumb input) then retuned again and the sound worked. In regard to remote, it uses the default with v1.5 or 2.0 Slingplayer, and Gerrys version with Slingerplayer.

**Reply by mike-uk** [reply]
Thanks bob after rescanning the sound come back and the video also play without stopping now.

**Comment by B-Junk** [comment]
I'm not sure if I am following all of this, maybe someone could just take a moment to explain how to scan for North American HDTV channels in a little more detail.  Here's what I've done:
1. Installed SlingPlayer1.5.1.338 on a computer and connected to my Slingbox with admin password.  
2. Set it to use a "External Box" channel with output on channel 58 (I happen to have a UHF analog modulator that outputs on UHF).  Works good on that single channel.  I also have a working TV antenna that receives 27 HDTV channels connected to the same coax.  A TV connected will scan all of these channels just fine.
3. Updated firmware to "SbPbProHdFwV2_01_420_HwV1_0_0Current.bin" by obtaining that file, putting it in the root SlingPlayer directory, and using SlingPlayer1.5.1.338 "Slingbox Configration" to "Update Slingbox Firmware".  Now indicated as 2.1.420 on "Information" tab.
4. Disconnected.  Installed "SlingPlayer_2.0.4522_Setup-Global.exe" on another computer.  Added the IP / port / admin / password.  I can see my channel 58 on it OK.
5. Launched "Setup Assistant" on SlingPlayer_2.0.4522.  It tries for a little while and then says "This feature requires an Internet Connection".  That's where I get stuck.  Again, I can see my channel 58 (analog) video OK. 

That's as far as I can get.  Any ideas on how to get past that point?

**Comment by sp222222** [comment]
I have
some notes on it. I think the certificate ran out in January as I cannot
get these other ProHDs scanned in.
